compiler/AST/ParamForLoop.cpp

```diff
@@ -499,6 +500,106 @@ CallExpr* ParamForLoop::foldForResolve()
       }
     }
   }
+  } else {
```

Comment:
Future work: check `paramMap` whether it maps any of low bound, high bound, stride. This way a param loop can reference param things, not just literals.

OTOH since we call chpl_compute_low_param_loop_bound and chpl_compute_high_param_loop_bound, maybe this is already taken care of.

Reply:
Good thought.  It is is taken care of.  I just extended the test to include a case like this.